src/Algebra/Definitions/RawMagma.agda

```diff
@@ -10,11 +10,11 @@
 
 {-# OPTIONS --cubical-compatible --safe #-}
 
-open import Algebra.Bundles using (RawMagma)
+open import Algebra.Bundles.Raw using (RawMagma)
```

Comment:
Definitely in favour of such localisation where possible, given my various recent issues around `RawX` vs `IsX` dependency in the hierarchy generally...